rust-toolchain.toml

```
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.77.0"
+components = [ "rustfmt", "rustc-dev" ]
+targets = [ "wasm32-unknown-unknown" ]
+profile = "minimal"
```

Not sure if you actually need `rustc-dev`, but `rustc-docs` and `clippy` are probably needed (`rustfmt` is used from nightly, so useless here), so either this:
```suggestion
components = [ "clippy", "rust-docs" ]
targets = [ "wasm32-unknown-unknown" ]
profile = "minimal"
```

Or probably just this if not wanting to mess with exact list too much:
```suggestion
targets = [ "wasm32-unknown-unknown" ]
profile = "default"
```